src/components/overlayCookie/OverlayCookie.js

```diff
@@ -51,9 +52,9 @@ class OverlayCookie extends Component {
           <p className={styles.cookieText}>
             {t('cookie-notification') + ' '}
             <span>
-              <a href="#cookies" onClick={() => onReadMore('cookies')}>
+              <Button size="small" type="tertiary" onClick={() => onReadMore('cookies')}>
                 {t('cookie-link')}
```

Comment:
Could we please make the COOKIES POLICY text title case => Cookies Policy?

Reply:
Ok I will do that

Reply:
I added a new prop to the button component called `typography`(couldn't think of a better name) that we can use to set the `text-transform` rule, it accepts `capitalize, uppercase or lowercase`. I dunno if you guys are  fine with this or you have an different solution.